engine/verification/fetcher/engine.go

```diff
@@ -228,12 +228,19 @@ func (e *Engine) processAssignedChunk(chunk *flow.Chunk, result *flow.ExecutionR
 // HandleChunkDataPack is called by the chunk requester module everytime a new requested chunk data pack arrives.
 // The chunks are supposed to be deduplicated by the requester.
 // So invocation of this method indicates arrival of a distinct requested chunk.
-func (e *Engine) HandleChunkDataPack(originID flow.Identifier, chunkDataPack *flow.ChunkDataPack, collection *flow.Collection) {
+func (e *Engine) HandleChunkDataPack(originID flow.Identifier, chunkDataPack *flow.ChunkDataPack) {
 	lg := e.log.With().
 		Hex("origin_id", logging.ID(originID)).
-		Hex("collection_id", logging.ID(collection.ID())).
 		Hex("chunk_id", logging.ID(chunkDataPack.ChunkID)).
 		Logger()
+
+	if chunkDataPack.Collection != nil {
+		// non-system chunk data packs have non-nil collection
```

Comment:
better to use else to indicate it's a system chunk in the log

Reply:
fixed in https://github.com/onflow/flow-go/pull/968/commits/042ff8feb53bda3b4b218992de77378c91121272